uiServices/src/testCaseServices.go

```diff
@@ -113,4 +127,63 @@ func putTestCase(rw http.ResponseWriter, req *http.Request) {
 	}
 
 	rw.WriteHeader(http.StatusNoContent)
+}
+
+func getAllTestCases(rw http.ResponseWriter, req *http.Request) {
+
+	testCasePathDir := getTestCaseHeader(req)
+	if len(testCasePathDir) <= 1 {
+		logrus.Error("No file directory entered")
+		rw.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	files, err := ioutil.ReadDir(testCasePathDir)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	testCases := make([]Case, 0)
+
+	for _, file := range files {
+		if filepath.Ext(testCasePathDir +file.Name()) == ".xml" {
+
+			testCase := new(testStrategies.TestDefinition)
+
+			filename := file.Name()
+
+			file, err := os.Open(fmt.Sprintf("%s%s", testCasePathDir, filename))
+			if err != nil {
+				continue
```

Comment:
I think that we should have log stmts here

Reply:
Also if there is an error we are not setting the response header

Reply:
I'll get to this soon. I'll add in log statements but jenny told me to leave out the response headers for each.